api/organisations/models.py

```diff
@@ -70,6 +70,15 @@ def over_plan_seats_limit(self):
             self.has_subscription() and 0 < self.subscription.max_seats < self.num_seats
         )
 
+    def over_plan_and_free_seats_limit(self):
```

Comment:
Can't we just refactor the method above to:

```python
def over_plan_seats_limit(self):
    return self.num_seats > getattr(self.subscription, "max_seats", MAX_FREE_SEATS)
```

This mew method feels like serious duplication.

Reply:
Tidied up

Reply:
What's the reason for not going with my suggestion? I don't really see the need for the new method at all...

Reply:
getattr doesnt work for our needs if the subscription object is missing but I've removed the additional method